include/xtensor/xview.hpp

```diff
@@ -1514,8 +1514,8 @@ namespace xt
     template <class... Args>
     inline auto xview<CT, S...>::make_index_sequence(Args...) const noexcept
     {
-        return std::make_index_sequence<(sizeof...(Args)+integral_count<S...>() > newaxis_count<S...>() ?
-                                         sizeof...(Args)+integral_count<S...>() - newaxis_count<S...>() :
+        return std::make_index_sequence<(sizeof...(Args)+integral_count<S...>() > newaxis_count<S...>() ? // here, "integral_count" and "newaxis_count"
+                                         sizeof...(Args)+integral_count<S...>() - newaxis_count<S...>() : // are taken into account (**label1**, see **label2** in xutils.hpp)
                                          0)>();
```

Comment:
Here, and index sequence is built, which takes new axis and selected dimensions into account... in `xutils.hpp/argument` this is not the case. Where - in our use case - this information about the `xview` is processed. I did not yet understand where the argument list (`Args...`) and/or the dimension information is transformed to prevent the error observed.